packages/editor/src/panels/hierarchy/helpers.ts

```diff
@@ -128,25 +139,32 @@ function buildHierarchyTree(
     isCollapsed: !getState(HierarchyTreeState).expandedNodes[sceneID]?.[entity],
     children: [],
     isLeaf: !(node.children && node.children.length > 0),
-    lastChild: lastChild
+    lastChild: lastChild,
+    isRendered: isRendered
   }
   array.push(item)
 
-  if (hasComponent(entity, GLTFComponent) && showModelChildren) {
-    const scene = GLTFComponent.getInstanceID(entity)
+  if (hasComponent(entity, ModelComponent) && showModelChildren) {
```

Comment:
please revert. we no longer use the ModelComponent, it is deprecated and only exists for comparison.

Reply:
roger that

Reply:
done